lib/fizzy/execute.cpp

```diff
@@ -127,6 +127,31 @@ uint64_t eval_constant_expression(ConstantExpression expr,
         return globals[global_idx - imported_globals.size()];
 }
 
+bool invoke_function(
```

Comment:
I think this should be grouped with `branch()`. Which is actually above the instantiate-helpers.

Reply:
Should I better bring `branch` down here?

Reply:
Probably, but would prefer that as a separate reordering commit. (We need to be more thorough reviewing so we don't end up needing to move things around all the time.)

Reply:
Added it after branch in "implement commit" and added another reorder commit.

Reply:
This should return `true` on success. Compare with `load_from_memory()`.